jetty9-base/src/main/jetty-base/etc/java-util-logging.properties

```diff
@@ -0,0 +1,9 @@
+.level = INFO
+io.netty.level = INFO
+io.gcr.level = INFO
```

Comment:
Why do we need to set log levels for netty and gcr here if we have the flushing guard in TracingLogHandler.publish()?